src/admin/items/items.service.ts

```diff
@@ -483,13 +484,28 @@ export class ItemsService {
 		newItemExternalId: string
 	) {
 		try {
+			const usersWithBothBookmarks = (
+				(await performQuery(
+					{
+						query: GET_USERS_WITH_BOTH_BOOKMARKS,
+						variables: {
+							oldItemUid,
+							newItemUid,
+						},
+					},
+					'data.users_profiles',
+					'Failed while checking users with both bookmarks.'
+				)) as Pick<Avo.User.Profile, 'id'>[]
+			).map((profile) => profile.id);
+
 			const response = await dataService.mutate({
 				mutation: REPLACE_ITEM_IN_COLLECTIONS_BOOKMARKS_AND_ASSIGNMENTS,
 				variables: {
 					oldItemUid,
 					oldItemExternalId,
 					newItemUid,
 					newItemExternalId,
+					usersWithBothBookmarks,
```

Comment:
should i still see both videos in my bookmarks, or should only one bookmark exist?
![image](https://user-images.githubusercontent.com/1710840/195296743-92aa03ef-4ab6-422b-9a3b-ced724e5d458.png)

currently i see both after these steps:
* bookmark both videos (afgaanse meisjes, salman rushdie)
* depublish and replace first video (afgaanse meisjes) with second (salman rushdie)

is het verwachte resultaat?
* ik zie 1 video bookmark van video 2 (salman rushdie)
* of is het huidige gedrag correct? ik zie beide videos waar de eerste (afgaanse meisjes) redirect naar de tweede (salman rushdie)

Reply:
https://meemoo.atlassian.net/browse/AVO-2181?focusedCommentId=35317

Reply:
ok, dan mergen we deze al, en zetten de jira taak op blocked?